### ISSUE TITLE
nginx: [emerg] mkdir() "/tmp/proxy_temp" failed (13: Permission denied)

### ISSUE THREAD

**Issue by stumpylog** [nginxinc/docker-nginx-unprivileged#141]
### Describe the bug

Starting with `ghcr.io/nginxinc/nginx-unprivileged:1.23.4`, permissions on /tmp appear to be incorrect somehow.  Returning to `ghcr.io/nginxinc/nginx-unprivileged:1.23.3` resolves the issue.

### To reproduce

```
docker run --rm -it --entrypoint=/bin/sh ghcr.io/nginxinc/nginx-unprivileged:1.23.3
$ whoami
nginx
$ id -u
101
$ id -g
101
$ touch /tmp/test
$ exit
```
versus
```
docker run --rm -it --entrypoint=/bin/sh ghcr.io/nginxinc/nginx-unprivileged:1.23.4
$ whoami
nginx
$ id -u
101
$ id -g
101
$ touch /tmp/test
touch: cannot touch '/tmp/test': Permission denied
$ exit
```

### Expected behavior

Files should be creatable in the `/tmp` directory

### Your environment

- Version of Docker: Server Version: 23.0.4
- Version of the NGINX Unprivileged Docker image: 1.23.3, 1.23.4, 1.24.0
- Target deployment environment/platform: Ubuntu 22.04.2, x86_64

### Additional context

Nothing comes to mind.

**Comment by alessfg** [comment]
Interesting. As far as I know nothing has changed between releases but I'll do some digging.

**Comment by stumpylog** [comment]
I've tracked it down a little further to the upstream debian slim image.  With the particular version I'm guessing was included in the .3 as the base, it's fine, with newer versions its not.  But also only on 1 of my computers.

So it seems unlikely to be this image's issue, but either upstream or maybe even some Docker oddity, since I would expect the same command/image to do the same thing regardless of computer.

**Comment by alessfg** [comment]
You are absolutely right, Docker behaving differently on different computers sort of defeats the purpose of Docker. Might be worth opening an issue in the Docker/Moby repo?

**Comment by stumpylog** [comment]
Yep, I've opened one up.  Feel free to close this out
